### ISSUE TITLE
Cards not exposed to color schemes in Basis

### ISSUE THREAD

**Issue by docwilmot** [backdrop/backdrop-issues#6239]
## Description of the bug

This is Basis `color.inc` list of CSS files it parses for color matching. Note no `cards.css`:
```
  'css' => array(
    'css/base.css',
    etc
    'css/skin.css',
  ),
```
Also there are no defaults for card colors. This means that the card colors are not colorizable

## Actual behavior

There are no color settings for card elements.

## Expected behavior

There should be, especially since cards are now a prominent front page feature.

## Additional information

Add any other information that could help, such as:
- Backdrop CMS version:
- Web server and its version:
- PHP version: 
- Database sever (MySQL or MariaDB?) and its version:
- Operating System and its version:
- Browser(s) and their versions:

**Comment by olafgrabienski** [comment]
I agree, it would be good to consider card colors in color schemes. That said, there are only two card color definitions in [`cards.css`](https://github.com/backdrop/backdrop/blob/1.x/core/themes/basis/css/component/cards.css):

```
.block-views-promoted-cards-block {
  background-color: #f2f2f2;
}
.view-promoted-cards .views-grid-box {
  background-color: #ffffff;
}
```

The more colorful elements (example below) are images and can only be changed by using different images. We could however reconsider the use of transparent images and CSS, see the original issue, e.g. at https://github.com/backdrop/backdrop-issues/issues/4903#issuecomment-1042788494 and below.

---

Card image example: 
<img src="https://github.com/backdrop/backdrop-issues/assets/14188307/1a6a26c5-995d-4fdb-ac4a-795b54a46ed9" width="400">

**Comment by NormPlum** [comment]
I've made a PR for this, but it should be used with the PR for https://github.com/backdrop/backdrop-issues/issues/6506 since that's required in order to see the background color (as it'll use icons instead of images with their own background color).

**Comment by NormPlum** [comment]
Here's how the cards look with both PRs applied:

<img width="1280" height="600" alt="Image" src="https://github.com/user-attachments/assets/1a30de3c-2c52-48a3-a2a7-bebb2989c79d" />
